cvat/apps/iam/urls.py

```diff
@@ -35,15 +35,41 @@
 if settings.IAM_TYPE == "BASIC":
     urlpatterns += [
         path("register", RegisterViewEx.as_view(), name=BASIC_REGISTER_PATH_NAME),
+    ]
+
+    password_change_view = PasswordChangeView
+
+    if "cvat.apps.api_tokens" in settings.INSTALLED_APPS:
+        import importlib
+
+        from rest_framework.decorators import authentication_classes
+
+        from cvat.apps.api_tokens.authentication import ApiTokenAuthentication
+
+        no_api_token_auth_classes = []
+        for auth_class_path in settings.REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"]:
+            auth_class_module_path, auth_class_name = auth_class_path.rsplit(".", maxsplit=1)
+            auth_class_module = importlib.import_module(auth_class_module_path)
+            auth_class = getattr(auth_class_module, auth_class_name)
```

Comment:
Seems like this could be replaced by `import_string` from `django.utils.module_loading`.